src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -340,7 +340,7 @@ function MoneyRequestConfirmationList({
 
     const shouldDisplayMerchantError = isMerchantRequired && (shouldDisplayFieldError || formError === 'iou.error.invalidMerchant') && isMerchantEmpty;
 
-    const isCategoryRequired = canUseViolations && !!policy?.requiresCategory;
+    const isCategoryRequired = !!policy?.requiresCategory;
```

Comment:
Removing `canUseViolations` will affect on 3 places as I explained [here](https://github.com/Expensify/App/issues/40591#issuecomment-2084495544), So we need to keep using it in `rightLabel` [here](https://github.com/Expensify/App/blob/8787a6464e6d2f75bff5433be63b3c663431c4fb/src/components/MoneyRequestConfirmationList.tsx#L873) and [here](https://github.com/Expensify/App/blob/8787a6464e6d2f75bff5433be63b3c663431c4fb/src/components/MoneyRequestConfirmationList.tsx#L897) by adding `canUseViolations` condition to `rightLabel`, because there is no validation for required tags/categories before sending the API in [confirm method](https://github.com/Expensify/App/blob/a462d5854359850359f248aaec58ad4bc2c0efcd/src/components/MoneyRequestConfirmationList.tsx#L591) and maybe also in BE (we may need to fix it later or in a new ticket)